catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-sbom-to-pyxis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
+  params:
+    - name: imageURL
+      description: URL of the main binary image.
+      type: string
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  steps:
+    - name: download-sbom-for-image
+      # Image taken from https://github.com/sigstore/cosign/pkgs/container/cosign%2Fcosign
+      image:
+        ghcr.io/sigstore/cosign/cosign:03468a1498876ccfb72e06f63fdb4f1c562dbca1
+      args:
+        - download
+        - sbom
+        - --output-file
+        - /workdir/sbom-cyclonedx.json
+        - $(params.imageURL)
```

Comment:
This should be quoted. At least I remember seeing it quoted in the docs. I could be wrong though.

Reply:
OK, fixed. I couldn't find anything about this in the doc (I didn't try very hard), but I did find examples of such variables being quoted in the docs.